### DISCUSSION TITLE
Intellij Namespace errors fix

### DISCUSSION THREAD

**Discussion by aeropagz** [wimdeblauwe/htmx-spring-boot#68] [Q&A]
Hello all,
is there a way to fix namespace errors in template in Intellij? Or are you just ignoring it?
I using using the htmx starter version 1.0.0 with spring boot 2.7.13 and Intellij version 2023.2.1.

**Comment by wimdeblauwe** [comment]
I usually just apply the quick-fix that IntelliJ provides to add this namespace: `xmlns:hx="http://www.w3.org/1999/xhtml"` resulting in something like this:

```
<!DOCTYPE html>
<html lang="en"
      xmlns:th="http://www.thymeleaf.org"
      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
      xmlns:hx="http://www.w3.org/1999/xhtml"
      layout:decorate="~{layout/main}" >
```
